jerry-core/ecma/base/ecma-err-msg.inc.h

```diff
@@ -0,0 +1,856 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
The file name could be `ecma-error-messages.inc.h` ( and the related ini file also). I don't see any reason why not to type it out.